CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# Release 1.32.0 30-04-2019
+
+- fix memory leak in percent_encode (#592)
+- Percent encode for grpc tailer: grpc-message (#588)
+- Polish the Check response error messages (#587)
+- Fix grpc_backend_ssl server_config format (#585)
+- Add 4 grpc_backend_ssl flags (#584)
+- Support SSL for gRPC backend (#582)
+-  Handle API_TARGET_BLOCKED and RESOURCE_EXHAUSTED (#581)
+- Override Compute Platform in serverless docker image (#579)
+- Fix the crash from undefined reference in start_esp.py (#578)
+
```

Comment:
Please only keep line 4, 8, 9, 10

Please fix indentation for line 9

Reply:
done.